app/src/main/java/de/danoeh/antennapod/fragment/actions/FeedMultiSelectActionHandler.java

```diff
@@ -126,6 +126,33 @@ private void keepUpdatedPrefHandler() {
         preferenceSwitchDialog.openDialog();
     }
 
+    private void skipInboxPrefHandler() {
+        PreferenceListDialog preferenceListDialog = new PreferenceListDialog(activity,
+                "Skip Inbox");
+        String[] items = activity.getResources().getStringArray(R.array.spnSkipInboxItems);
+        String[] values = activity.getResources().getStringArray(R.array.spnSkipInboxValues);
+        preferenceListDialog.openDialog(items);
+        preferenceListDialog.setOnPreferenceChangedListener(which -> {
+            FeedPreferences.SkipInboxSetting skipInboxSetting = null;
+            switch (values[which]) {
+                case "global":
+                    skipInboxSetting = FeedPreferences.SkipInboxSetting.GLOBAL;
```

Comment:
There are quite many uses of this string. How about adding a string value to the enum? Something like `FeedPreferences.SkipInboxSetting.GLOBAL.value()`

Reply:
I have done this, however I can't use this `value()` method from the enum in the `switch` statement, because I get an error that `case` statements require constant values.

Reply:
Oh, hmm, then maybe some `public static final String` somewhere?

Reply:
I was able to solve this by adding static string attributes to the enum and then passing them to the constructor. Since the auto-delete preference followed the same design pattern (it's what I based my new preference on), and there were duplicated strings across the codebase for auto-delete, I made the same changes to that enum.

Here's the specific change that I made: https://github.com/AntennaPod/AntennaPod/blob/faa267f4/model/src/main/java/de/danoeh/antennapod/model/feed/FeedPreferences.java#L18-L42